crates/agglayer-config/src/grpc.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+
+pub const DEFAULT_GRPC_MESSAGE_SIZE: usize = 64 * 1024 * 1024; // 64MB
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+pub struct GrpcConfig {
+    #[serde(
+        skip_serializing_if = "same_as_default_max_decoding_message_size",
+        default = "default_max_decoding_message_size"
+    )]
+    pub max_decoding_message_size: usize,
+    #[serde(
+        skip_serializing_if = "same_as_default_max_encoding_message_size",
+        default = "default_max_encoding_message_size"
+    )]
+    pub max_encoding_message_size: usize,
```

Comment:
Would be nice to use some sort of [human-readable format](https://crates.io/crates/byte-unit) for the size specification. The config values here will be huge numbers which are rather difficult to read.